spec/index.html

```diff
@@ -6813,13 +6817,17 @@ <h5>ENCODE_FOR_URI</h5>
           </section>
           <section id="func-concat">
             <h5>CONCAT</h5>
-            <pre class="prototype nohighlight"><span class="return">string literal</span>  <span class="operator">CONCAT</span>(<span class="type">string literal</span> <span>ltrl<sub>1</sub></span> ... <span class="type">string literal</span> <span>ltrl<sub>n</sub></span>)</pre>
-            <p>The <code>CONCAT</code> function corresponds to the XPath <a data-cite="XPATH-FUNCTIONS#func-concat">fn:concat</a> function. The function accepts string
```

Comment:
As this doesn't rely on `fn:concat` anymore, we may want to add some details regarding the handling of unicode, which were implicit due to the reliance on `fn:concat`.

Specifically, I'm referring to the notes in https://www.w3.org/TR/xpath-functions/#func-concat